plugins/rich-editor/README.md

```diff
@@ -0,0 +1,218 @@
+# Rich Editor (WIP)
+
+A highly-functional WYSIWYG editor for Vanilla built on top of [contenteditable](https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Editable_content) using [Quill.js](https://quilljs.com/), [Quill's document model parchment](https://github.com/quilljs/parchment) and [React](https://reactjs.org/).
+
+This plugin conflicts with and aims to replace some existing plugins.
+
+- [Advanced Editor](../editor)
+- [Quotes](../Quotes)
+
+## Core features
+
+- Consistent WYSIWYG experience.
+- Rich text embeds of external and internal content.
+- Improved video and image embedding.
+- Intuitive minimal design.
+- Opinionated restrictions on functionality.
+- Consistent serialized format that can be easily parsed on the server and client (JSON).
+- Convert all existing Advanced Editor formats (HTML/Wysiwyg, Markdown, BBCode, Text, TextEx).
+- Extendable to support future embeddable objects.
+
+## Formatting Options
```

Comment:
You shouldn't have an empty h2 here. If the following heading is a sub section of "Formatting Options" it should go down a level.